src/Components/Navbar/Navbar.jsx

```diff
@@ -30,23 +35,22 @@ export const Navbar = () => {
             />
           </a>
           <div class="badges">
-            {/* <Link to="/Wishlist"> */}
+            <Link to="/Wishlist">
```

Comment:
link path should be in small case.  eg. "/wishlist". Capital case path are not suggested to be used.